### PR TITLE
Feat: add Activity Page and Top Affiliate Page and Analytic Page

### PR DESCRIPTION
**Pull request by louisdevzz**

### CODE REVIEW

File: apps/BosGenaDrop/widget/Mintbase/App/Activity/Activity.jsx [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+        description: "4,644,3773"
+    },
+    {
+        title: "Total Transactions (All Near)",
+        description: "4,644,3773"
+    },
+    {
+        title: "Total Transactions (All Near)",
+        description: "4,644,3773"
+    },
```
Comment (Ebube111) [inline apps/BosGenaDrop/widget/Mintbase/App/Activity/Activity.jsx:256]: All these are dummy data, could you reach out to mintbase team and request the API being called on that page. Let me know if you need help reaching out

File: apps/BosGenaDrop/widget/Mintbase/App/Activity/Analytics.jsx [hunk 2]
```diff
@@ -0,0 +1,308 @@
...
+    },
+    {
+        title: "Listed",
+        description: "4,644,3773"
+    },
+    {
+        title: "Affiliate",
+        description: "4,644,3773"
+    },
+]
```
Comment (Ebube111) [inline apps/BosGenaDrop/widget/Mintbase/App/Activity/Analytics.jsx:292]: Request here as well

File: apps/BosGenaDrop/widget/Mintbase/App/Activity/Analytics.jsx [hunk 3]
```diff
@@ -0,0 +1,308 @@
...
+
+
+return(
+    <>
+        <SearchBar>
+            <div className="title">Months</div>
+            <input type="search" className="search"  placeholder="0"/>
```
Comment (Ebube111) [inline apps/BosGenaDrop/widget/Mintbase/App/Activity/Analytics.jsx:299]: is the searchbar integrated ? because it needs to be

**Review by Ebube111 — CHANGES_REQUESTED** [review]
Code looks good. Nice Job.
Look into the comments and reach out if you need help connecting to mintbase.

Also for the component `HomeTableActivity` you made changes there, please confirm that where it is being used, nothing is breaking, just to be sure. Thanks

File: apps/BosGenaDrop/widget/Mintbase/App/Activity/Activity.jsx (hunk 1 above)
Comment (louisdevzz) [reply apps/BosGenaDrop/widget/Mintbase/App/Activity/Activity.jsx:256]: I can't reach out to mintbase team, Can you reach out to mintbase and Can help me request the API?

File: apps/BosGenaDrop/widget/Mintbase/App/Activity/Analytics.jsx (hunk 3 above)
Comment (Jikugodwill) [reply apps/BosGenaDrop/widget/Mintbase/App/Activity/Analytics.jsx:299]: It means you need to connect that page to that Navbar, sho the Navbar shows when you view it @louisdevzz

Comment (louisdevzz) [reply apps/BosGenaDrop/widget/Mintbase/App/Activity/Analytics.jsx:299]: yes, thanks for answer

**Review by Ebube111 — APPROVED** [review]
Looks good

**Pull request merged**